apps/i18n/dance/en_us.json

```diff
@@ -27,6 +18,15 @@
   "danceFeedbackUseSetSize": "Use the `set backup_dancer2 size` block to make that dancer smaller.",
   "danceFeedbackUseSetTint": "Use the `set tint` block to change a dancer's tint.",
   "danceFeedbackUseStartMapping": "Try adding the <xml><block type=\"Dancelab_startMapping\"><title name=\"SPRITE\">right_unicorn</title><title name=\"PROPERTY\">\"scale\"</title><title name=\"RANGE\">\"bass\"</title></block></xml> block to your program.",
-  "danceFeedbackKeepCoding": "Keep coding! Something's not quite right yet.",
-  "measure": "Measure:"
+  "measure": "Measure:",
+  "DanceAiNewMove": "⚠️ You need to add a block to make the dancer do a new move.",
+  "danceAIFeedbackNeedNewDancer": "⚠️ You need to make a dancer.",
+  "DanceAiFeedbackAiBlockNoParameters": "⚠️ Choose 3 emojis for AI to generate effects.",
+  "DanceAiFeedbackAiBlockNotConnectedSetup": "⚠️ Connect the `AI generate effect` block to the `setup` block.",
+  "DanceAIFeedbackNoAiBlock": "⚠️ Add the `AI generate effect` block to setup.",
```

Comment:
Are we calling it the "AI generate effect" block?  Should it just be "generate effect" block?  If we do include AI, should it be "A.I." since we are presumably referring to the image of the bot?